### PR TITLE
[ruv] Add new extractor

### PR DESCRIPTION
**Pull request by orng**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [x] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information
Added new extractor for Icelandic national TV station Rúv's website ruv.is. In particular, ruv.is/sarpurinn which is where most shows are made available online.

### CODE REVIEW

File: youtube_dl/extractor/ruv.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._og_search_title(webpage)
+        self.report_extraction(video_id)
+        video_url = self._html_search_regex(r'video.src\s+=\s+["\'](.+?)["\']', webpage, u'video URL')
```
Comment (dstftw) [inline youtube_dl/extractor/ruv.py]: Dots should be escaped. Regex should be relaxed. No `u''`.

File: youtube_dl/extractor/ruv.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            'title': u'Fréttir',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._og_search_title(webpage)
+        self.report_extraction(video_id)
```
Comment (dstftw) [inline youtube_dl/extractor/ruv.py]: Remove.

**Comment by orng** [comment]
Thanks for the comments. I believe I have addressed them all in the two commit above. I have no idea why some of the tests are failing though, but as far as I can tell it's due to some issue unrelated to my changes, is that correct?

File: youtube_dl/extractor/ruv.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class RuvIE(InfoExtractor):
+    _VALID_URL = r'http?://(?:www\.)?ruv\.is/sarpurinn/ruv/\w+/(?P<id>[0-9]+)'
```
Comment (dstftw) [inline youtube_dl/extractor/ruv.py]: `http?` incorrect.

**Comment by dstftw** [comment]
Squash commits.

**Comment by orng** [comment]
I addedthe missing s in "http?" (should be "https?") in url regex and rebased into one commit. 
Hopefully this does it. Thanks for pointing out my mistakes.

**Pull request closed without merging**